config/crds/kyverno.io_clusterpolicies.yaml

```diff
@@ -2179,6 +2179,77 @@ spec:
                       type: string
                     type: array
                 type: object
+              conditions:
+                description: Conditions is a list of conditions that apply to the
+                  policy
+                items:
+                  description: "Condition contains details for one aspect of the current
+                    state of this API Resource. --- This struct is intended for direct
+                    use as an array at the field path .status.conditions.  For example,
+                    type FooStatus struct{     // Represents the observations of a
+                    foo's current state.     // Known .status.conditions.type are:
+                    \"Available\", \"Progressing\", and \"Degraded\"     // +patchMergeKey=type
+                    \    // +patchStrategy=merge     // +listType=map     // +listMapKey=type
+                    \    Conditions []metav1.Condition `json:\"conditions,omitempty\"
+                    patchStrategy:\"merge\" patchMergeKey:\"type\" protobuf:\"bytes,1,rep,name=conditions\"`
+                    \n     // other fields }"
```

Comment:
This (line 2187-2195) does not look properly formatted to me.

Reply:
These are the comments generated by `controller-gen` coming from the `metav1` package. Nothing we can really do about it I guess.

Reply:
Whoops, sorry for that.